docs_src/microservices/V2TopLevelMigration.md

```diff
@@ -0,0 +1,97 @@
+# V2 Migration Guide
+
+!!! edgey "EdgeX 2.0"
+    Many backward breaking changes occurred in the EdgeX 2.0 (Ireland) release which may require some migration depending on your use case.
+
+This section describes how to migrate from EdgeX 1.x to EdgeX 2.0 at a high level and refers the reader to the appropriate detail documents. The areas to consider for migrating are:
+
+- [Compose File](#compose-file)
+- [Database](#database)
+- [Custom Configuration](#custom-configuration)
+- [Custom Device Service](#custom-device-service)
+- [Custom Applications Service](#custom-applications-service)
+
+## Custom Compose File
+
+The compose files for V2 have many changes from their V1 counter parts. If you have customized a V1 compose file to add additional services or tweak overrides,  it is highly recommended that you start with the appropriate V2 compose file and re-add your customizations. It is very likely that the sections for your additional services will need to be migrated to have the proper environment overrides. Best approach is to use one of the V2 service sections that closest matches your service (App or Device) as a template.
+
+## Database
+
+There currently is no migration path for the data stored in the database. The V2 data collections are stored separately from the V1 data collections in the Redis database. Redis is now the only supported database, i.e. support for Mongo has been removed.
+
+!!! note
+    Since the V1 data and V2 data are stored separately, one could create a migration tool and upstream it to the EdgeX community.
+
+!!! warning
+    If the database is not cleared before starting the V2 services, the old V1 data will still reside in the database taking up useful memory. It is recommended that you first wipe the database clean before starting EdgeX 2.0 Services. That is unless you create a DB migration tool.
+
+The following sections describe what you need to be aware for the different services that use the database.
+
+### Core Data
+
+The Event/Reading data stored by Core Data is considered transient and of little value once it has become old. The V2 versions of these data collections will be empty until new Events/Readings are received from V2 Device Services. 
+
+### Core Metadata
+
+The data stored by Core Metadata will be recreated when the V2 versions of the Device Services start-up. The statically declared devices will automatically be created and device discovery will find and add existing devices. Any devices created manually via the V1 REST API will have to be recreated using the V2 REST API.
+
+### Core Command
+
+Core Command in V2 no longer uses the database. It now retrieves the needed information from Core Metadata REST APIs.
+
+### Support Notifications
+
+Any `Subscriptions` created via the V1 REST API will have to be recreated using the V2 REST API. The `Notification` and `Transmission`collections will be empty until new notifications are sent using EdgeX 2.0 
```

Comment:
We need a migration guide for subscriptions that this guide references.

Reply:
Agreed. I can put a place holder.

Reply:
Actually, how does this differ from any of the other data? Are you wanting a description of converting V1 model to V2 DTO?